src/views/sandwich-view.tsx

```diff
@@ -18,7 +18,53 @@ interface SandwichViewProps {
   glCanvas: HTMLCanvasElement
 }
 
-class SandwichView extends StatelessComponent<SandwichViewProps> {
+interface SandwichViewState {
+  isDragging: boolean
+  resizerPos: number | null
+  initialPos: number
+  delta: number
+}
```

Comment:
Component state disappears when components unmount, which in the case of speedscope, will happen when you e.g. switch from sandwich view to another view. I think it would be helpful to persist resize state across view switches so you don't need to keep resizing it when you switch between views.

To deal with application-level state, speedscope uses redux (https://redux.js.org/). We store all of the application state in a single state object that lives longer than any of the components we unmount in speedscope's application lifecycle.

This means that rather than having the resize position stored in the `state` property of the `SandwichView`, it needs to be stored in the `SandwichViewState` object specified here: https://github.com/jlfwong/speedscope/blob/30ca6291ca93557e2bb77a2bafc5bac580627102/src/store/sandwich-view-state.ts#L10-L12

If you look at the diff I put in the top-level review feedback comment, you can see how I did that.

This part of the architecture of speedscope is admittedly not very well documented :)